src/RefractAST.cc

```diff
@@ -719,15 +670,30 @@ namespace drafter
                 throw std::runtime_error("Unhandled type of DataStructure");
         }
 
+        return element;
+    }
+
+    sos::Object SerializeRefract(refract::IElement* element, const refract::Registry& registry) {
+
         if (!element) {
             return sos::Object();
         }
 
+        refract::ExpandVisitor expander(registry);
```

Comment:
The duplication happens when I add boutique. We will leave it as it is now. But we should come back to this later and see if this needs reorganization.